data/perform-sql-updates.sh

```diff
@@ -24,7 +24,7 @@ echo "Generating functions from YAML..."
 
 # store SQL in a temporary file, so make a temporary directory for it.
 tmpdir=$(mktemp -d "${TMPDIR:-/tmp/}$(basename 0).XXXXXXXXXXXX")
-sqlfile="${tmpdir}/generated-fuctions.sql"
+sqlfile="${tmpdir}/generated-functions.sql"
```

Comment:
Does this spelling fix need to be applied anywhere else?

Reply:
> Does this spelling fix need to be applied anywhere else?

I don't think so since I searched for `fuctions` in tileopes, tilequeue, didn't find any references.

Reply:
✅ Looks like we can call it whatever we want because a few lines down the reference is used a tmp (burner) file to get things into Postgres with more observability.